src/assets/data/programs.json

```diff
@@ -6,4 +6,11 @@
         "url": "https://hacktoberfest.digitalocean.com/",
         "end_date": "2022-10-31"
     }
-]
\ No newline at end of file
+    {
+        "title": "MLH Fellowship 2022",
+        "image": "https://cdn-images-1.medium.com/max/1200/1*GfbRjYz2qvPk0-GxF9O6DQ.jpeg"
+        "description": "A fully remote, 12-week internship where participants earn a stipend and learn to collaborate on real open source projects with peers and engineers from top companies."
+        "url": "https://fellowship.mlh.io/#programs"
+        "end_date": "throughout the year"
```

Comment:
@Sidd2024, Thanks for the contribution. I think we should specify the end date as null or a certain date which will be better here.